netlify.toml

```diff
@@ -1,3 +1,3 @@
 [build]
   publish = "_site"
-  command = "npm run build"
+  command = "eleventy"
```

Comment:
Instead of changing this to an overt command, we should update the `npm run build` script in `package.json` to run `eleventy`. This will also make sure that the local Eleventy is run, since `npm` scripts always prefer the local version.

Reply:
Updated